Backpack/Card/Classes/BPKCard.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * `BPKCard` is a subclass of `UIControl` which contains the Skyscanner card style.
+ * Cards are a lightweight control with a shadow.
+ * Cards can be configured to have padding or not.
+ */
+IB_DESIGNABLE @interface BPKCard: UIControl
+
+
+/**
+ * Whether the card should provide padding or not.
+ */
+@property (nonatomic, assign, getter=isPadded) IBInspectable BOOL padded;
+
+- (instancetype)initWithPadded:(BOOL)padded NS_DESIGNATED_INITIALIZER;
+- (instancetype)initWithFrame:(CGRect)frame NS_UNAVAILABLE;
```

Comment:
Don't mark this and `new` as unavailable they are needed for IB_DESIGNABLE to work. Please ensure they are implemented and set up the component in a default state